vod/media_set.h

```diff
@@ -146,6 +157,11 @@ typedef struct {
 
 	media_notification_t* notifications_head;
 
+	media_closed_captions_t* closed_captions;
+	media_closed_captions_t* closed_captions_end;
+	uint32_t closed_captions_count;
```

Comment:
can be removed, the code only does closed_captions_count > 0, and it can do closed_captions < closed_captions_end instead